### DISCUSSION TITLE
This app seems to be dead, nobody is addressing issues. Get your money out and RUN!

### DISCUSSION THREAD

**Discussion by SwissMaWi** [DeFiCh/app#1272] [General]
This app seems to be dead, nobody is addressing issues. Get your money out and RUN!

**Comment by John-Gee** [comment]
I fear the only wallet work is now on the mobile version.

**Reply by SwissMaWi** [reply]
I am pulling out of Defi. The coin is currently crashing and soon nobody will be able to get the money out anymore. Thanks to all and byebye

**Reply by beoschmann** [reply]
It's been hard, had to try dozens of times for few days, install it on several versions and mac/windows to finally been able to change my coins and send them away ! The app is so buggy that's a nightmare...
